feature-libs/order/components/order-details/order-details.module.ts

```diff
@@ -70,21 +71,45 @@ const moduleComponents = [
       cmsComponents: {
         AccountOrderDetailsActionsComponent: {
           component: OrderDetailActionsComponent,
+          guards: [AuthGuard],
         },
         AccountOrderDetailsItemsComponent: {
           component: OrderDetailItemsComponent,
+          guards: [AuthGuard],
+          data: {
+            enableAddToCart: true,
+          },
+        },
+        AccountOrderDetailsGroupedItemsComponent: {
+          component: OrderDetailItemsComponent,
+          guards: [AuthGuard],
           data: {
             enableAddToCart: true,
+            displayConsignmentDelivery: true,
           },
         },
         AccountOrderDetailsTotalsComponent: {
           component: OrderDetailTotalsComponent,
+          guards: [AuthGuard],
+        },
+        AccountOrderDetailsOverviewComponent: {
+          component: OrderOverviewComponent,
+          guards: [AuthGuard],
         },
-        AccountOrderDetailsShippingComponent: {
-          component: OrderDetailShippingComponent,
+        AccountOrderDetailsSimpleOverviewComponent: {
+          component: OrderOverviewComponent,
+          guards: [AuthGuard],
+          data: {
+            simple: true,
+          },
         },
+        // AccountOrderDetailsBillingComponent: {
+        //   component: OrderDetailBillingComponent,
+        //   guards: [AuthGuard],
+        // },
```

Comment:
Is this dead code or something that should have been uncommented?  Either remove the dead code or uncomment it.

Reply:
We have cms component `OrderDetailBillingComponent`, but in the UI design, the billing is part of overview. I will remove it.

Reply:
removed